### PR TITLE
Update 1.1 CN translations

### PR DESCRIPTION
**Pull request by hedw1gP**

### CODE REVIEW

File: help/zh_CN/commands.yml [hunk 1]
```diff
@@ -1,138 +1,148 @@
...
-            
+
             remove:
                 |
                     {l}{6}/{r}camera path {8}remove{r} {7}<index> <point_index>{r}
-                    
-                    {r}在路径索引号 {7}<index>{r} 的路径里移除路径点索引号 {7}<point_index){r}。
\ No newline at end of file
+
+                    {r}在路径索引号 {7}<index>{r} 的路径里移除路径点索引号 {7}<point_index){r}。
```
Comment (mchorse) [inline help/zh_CN/commands.yml]: Hm, this line is a bit weird. I think it should be `<point_index>` instead of `<point_index)`.

**Comment by mchorse** [comment]
Great job! 🙂 
I'll wait for README to be translated too, before merging. By the way, I'm going to also work on Wiki, do you need to translate it too? It will have some important sections like math modifier usage (for math expressions).

**Comment by hedw1gP** [comment]
I will translate Wiki too(after you push the wiki), and this PR is ready to merge

**Comment by mchorse** [comment]
Alright, thank you very much! 🙂

**Pull request merged**